extensions/cognigy/src/knowledge-connectors/chunker/simpleSplit.ts

```diff
@@ -0,0 +1,70 @@
+/* Node Modules */
+import { TokenTextSplitter, TokenTextSplitterParams, CharacterTextSplitter, CharacterTextSplitterParams } from "langchain/text_splitter";
+import * as modelToEncoding from '@dqbd/tiktoken/model_to_encoding.json';
+import { TiktokenEncoding, TiktokenModel } from "@dqbd/tiktoken";
+
+// Max chunk size limit
+const MAX_CHUNK_SIZE = 2000;
+
+/**
+ * This method resolves the proper encoding name for a given language model.
+ * It's a helper function to find the proper encoding name for a model without looking it up.
+ * The mapping comes from the "@dqbd/tiktoken" package, which is internally used by langchain
+ *
+ * @param modelName a tiktoken-compatible model name
+ * @returns a tiktoken-compatible encoding name
+ */
+const getEncodingNameForModelName = (modelName: TiktokenModel): TiktokenEncoding => {
+    const encodingName = modelToEncoding[modelName] as TiktokenEncoding;
+    if (!encodingName)
+        return null;
+    return encodingName;
+};
+
+interface ISimpleSplitOptions extends TokenTextSplitterParams {};
+const TokenSplitterDefaultOptions: Partial<ISimpleSplitOptions> = {
+    chunkSize: 512,
+    chunkOverlap: 128,
+    encodingName: getEncodingNameForModelName('gpt-3.5-turbo')
+};
+
+interface ITextSplitOptions extends CharacterTextSplitterParams {};
+const CharSplitDefaultOptions: Partial<ITextSplitOptions> = {
```

Comment:
`ITextSplitOptions` is not used anywhere then the next line. I suggest simplifying it

```suggestion
const CharSplitDefaultOptions: Partial<CharacterTextSplitterParams> = {
```